src/menu/menu.scss

```diff
@@ -19,13 +19,14 @@
   margin: 0;
   padding: 0;
 
-  > li {
+  > li:not(.iui-divider) {
     display: flex;
     align-items: center;
     font-size: $iui-font-size;
-    padding: 0 $iui-sm;
+    padding: $iui-component-padding-vertical $iui-sm;
```

Comment:
How come it was 0 before?

Reply:
Reverted back to 0, using component height vars instead.  Does now require `.iui-large` to be applied to enable the secondary line within a menu item.